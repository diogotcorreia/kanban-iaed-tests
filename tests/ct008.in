t 1 a
t 3 a
t -2 b
t 0 c
t 2 b
l
l 3
l 2 1
l 1 2
l 45
l 56 2 -3
n 3
n 0
n -2
n -23
n 34
u ola
u       testes
u
u ola
u
m 1 ola TO DO
m 5 ola DONE
m 1 ola DONE
m 1 ola TO DO
m 1 nao IN PROGRESS
m 1 ola DEBUG
m 5 bruh DEBUG
m 2 testes IN PROGRESS
n 45
m 2 testes TO DO
m 2 testes DONE
d DONE
d DEBUG
a
a DONE
a debug
a 123
a
q
