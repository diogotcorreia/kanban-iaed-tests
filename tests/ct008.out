task 1
duplicate description
invalid duration
invalid duration
task 2
1 TO DO #1 a
2 TO DO #2 b
3: no such task
2 TO DO #2 b
1 TO DO #1 a
1 TO DO #1 a
2 TO DO #2 b
45: no such task
56: no such task
2 TO DO #2 b
-3: no such task
3
3
invalid time
invalid time
37
ola
testes
user already exists
ola
testes
no such task
duration=0 slack=-1
task already started
no such user
no such activity
no such task
82
task already started
duration=45 slack=43
1 37 a
2 37 b
no such activity
TO DO
IN PROGRESS
DONE
duplicate activity
invalid description
TO DO
IN PROGRESS
DONE
123
